templates/profiles/membership.html

```diff
@@ -1,10 +1,36 @@
 {% load crispy_forms_tags %}
 
+
 <div class="row">
     <div class="col-xs-12 col-sm-6 col-md-4">
         <h3>Medlemskap</h3>
     </div>
 </div>
+
+{% if not user.first_name or not user.last_name %}
+<div class="row">
+    <div class="col-xs-12 col-sm-6 col-md-5">
+        <form id="user-name-form" method="post" action="{% url 'update_user_name' %}">
+            {% csrf_token %}
+            <h3>Vi trenger ditt fornavn og etternavn!</h3>
```

Comment:
Vi trenger jo bare dette for _manuell_ søknad—vi får det fra FEIDE ved automatisk